### PR TITLE
feat: Implement `plural upgrade`

### PR DESCRIPTION
**Pull request by michaeljguarino**
## Summary
creates an entry on an upgrade queue, especially helpful for CONFIG upgrades in CD

## Test Plan
created an upgrade locally

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/plural.go [hunk 1]
```diff
@@ -235,6 +235,18 @@ func (p *Plural) getCommands() []cli.Command {
...
+			Name:      "upgrade",
+			Usage:     "creates an upgrade in the upgrade queue QUEUE for application APP",
+			ArgsUsage: "QUEUE APP",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "f",
+					Usage: "file containing upgrade contents, use - for stdin",
+				},
+			},
+			Action: p.handleUpgrade,
```
Comment (zreigz) [inline cmd/plural/plural.go:248]: it would be good to add at least `requireArgs` validator

**Review by zreigz — CHANGES_REQUESTED** [review]

File: cmd/plural/plural.go (hunk 1 above)
Comment (michaeljguarino) [reply cmd/plural/plural.go:248]: oh good point

**Comment by michaeljguarino** [comment]
@zreigz ptal

**Review by zreigz — APPROVED** [review]
lgtm

**Pull request merged**